homeassistant/components/sensor/bme280.py

```diff
@@ -117,7 +117,7 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     except KeyError:
         pass
 
-    async_add_devices(dev)
+    async_add_devices(dev, True)
```

Comment:
Please remove the call to `update` from the `BME280Handler ` constructor.

Reply:
Line 129?

Reply:
I'm not sure if it's not needed to signal the underlying library the first reading.

Reply:
Resolved in chat. Update in constructor is not for an entity